deltachat-ffi/Cargo.toml

```diff
@@ -16,6 +16,7 @@ crate-type = ["cdylib", "staticlib"]
 
 [dependencies]
 deltachat = { path = "../", default-features = false }
+deltachat-jsonrpc = { path = "../deltachat-jsonrpc" }
```

Comment:
unless rust is smart enough to remove unused code somehow, we should create the whole json-rpc as an optional feature so that android/ios do not grow in size and things are mostly unchanged because of json-rpc in.

Reply:
right this adds ~3mb, though I'm not sure how we should make it conditionally, because if we exclude it by default nix, flatpack and co. with system installed libdeltachat might forget to build libdeltachat with the jsonrpc feature and then wonder why it doesn't compile.

So I think opt-out feature would be more manageable for us because we only need to set that option in android and iOS then.

Reply:
funfact: 3mb is also what we saved recently with the `panic=abort` setting